keyboards/handwired/illera16/config.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 y (@villeramassana)
```

Comment:
This file must be removed, and content migrated to info.json

Reply:
since I've deleted this file i'm not able to compile it

Reply:
That's because the contents need to be migrated over.

Reply:
This file **still** needs to be removed.